### PR TITLE
Readme cleanup

### PR DESCRIPTION
**Pull request by leakymemory**
Remove the no longer needed steps around the OneNoteApi and picker projects as well as the node server.

### CODE REVIEW

**Comment by mttwc** [comment]
Nice! :sheep:it

**Comment by larsendaniel** [comment]
:+1:  Hey, this is what I was going to do for my first PR!  Guess you beat me too it.  ;)

**Comment by VishaalK** [comment]
its lit

File: README.md [hunk 1]
```diff
@@ -9,31 +9,26 @@ $ npm install --global gulp
...
-
-### 4. Install the WebClipper packages
+### 3. Install the WebClipper packages
 From the root of this project, run:
 ```sh
 $ npm install
 ```
 
-### 5. Setup the external d.ts files and link the non-published npm packages
+### 4. Setup the external d.ts files and link the non-published npm packages
```
Comment (iconix) [inline README.md]: do we need this "non-published" adjective anymore?

Comment (iconix) [inline README.md]: Yeah, I think the phrase
> link the non-published npm packages
 was a reference to the old `gulp link` and `gulp unlink` commands and can be deleted now :)

File: README.md [hunk 2]
```diff
@@ -9,31 +9,26 @@ $ npm install --global gulp
...
 From the root of this project, run:
 ```sh
 $ gulp setup
 ```
 
-### 6. Build and Test
+### 5. Build and Test
 The default gulp command is to build and test:
 ```sh
 $ gulp
```
Comment (iconix) [inline README.md]: would be good to mention that `gulp` also runs TSLint and tests

**Comment by iconix** [comment]
Looks good :ship:

**Pull request merged**